### PR TITLE
bug 1402037 - improve shutdownhang signatures

### PR DESCRIPTION
**Pull request by willkg**
We have a number of shutdownhang signatures where the stack is rife with
"uninteresting" bits such that the signature gets truncated before getting to
something interesting or relevant or actionable.

This adds a bunch of things to the irrelevant list such that they don't show up
in the signature.

I did some regression testing and these strings only show up in shutdownhang
crashes. If that changes, we should rethink them on a string-by-string basis.

### CODE REVIEW

**Comment by willkg** [comment]
Examples of changes in crashes from this change here: https://bugzilla.mozilla.org/show_bug.cgi?id=1402037#c23

File: socorro/siglists/irrelevant_signature_re.txt [hunk 1]
```diff
@@ -37,9 +37,18 @@ mnt@asec@org\.mozilla\.f.*-\d@pkg\.apk@classes\.dex@0x.*
...
 mozilla::gfx::Log<.*
 mozilla::ipc::RPCChannel::Call
+mozilla::ThreadEventQueue.*
+nsEventQueue::GetEvent
 _NSRaiseError
+nsThread::GetEvent
+nsThread::nsChainedEventQueue::GetEvent
+nsThread::ProcessNextEvent
+NS_ProcessNextEvent
+NtWaitForKeyedEvent
```
Comment (willkg) [inline socorro/siglists/irrelevant_signature_re.txt]: I should collapse this one like the following one.

**Comment by willkg** [comment]
@lonnen If we can, I'd love to get this landed and tested before tomorrow. Then we can push it to -prod tomorrow.

**Comment by lonnen** [comment]
R+

**Comment by willkg** [comment]
Thank you!

**Pull request merged**